carbonplan_forests/plot/fire.py

```diff
@@ -126,3 +181,57 @@ def evaluation(data, model, data_var='vlf', model_var='prob', projection='albers
     )
 
     return chart
+
+
+def package_for_altair(fire, climate, label='value'):
```

Comment:
i don't see this function used anywhere, maybe drop from the PR...

Reply:
So, if I used these functions for analyses that didn't end up making it anywhere, but maybe I want to save them for future use- what's the best practice there? Just leave it on a miscellaneous branch? I guess it's stored in the commit history. I'll delete them for now.